activemq-broker/src/main/java/org/apache/activemq/replica/ReplicaRole.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.activemq.replica;
+
+public enum ReplicaRole {
+    source, replica, dual
```

Comment:
Is replica used? It looks like only source and dual are supported in `ReplicaPlugin`

Reply:
yes. it's used here: https://github.com/apache/activemq/pull/848/files#diff-7aeef5b9b38e464d9a560c33dff459f2fe95f8cb5cc0ab3d45424e0a50092fc8R56

plus we are working on real roles now, so it will be used more in the nearest future